packages/modules/itwin-error-handling-react/package.json

```diff
@@ -38,7 +38,7 @@
     "lib/**/*"
   ],
   "dependencies": {
-    "apollo-client": "^2.6.3",
+    "apollo-client": "^2.6.4",
```

Comment:
You accidentally bumped this, but wondering why we have a dep on this in the first place in this pkg